File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let sum = num_list.reduce((acc, curr) => acc + curr, 0) ** 2;
    let product = num_list.reduce((acc, curr) => acc * curr, 1);
    if(sum>product){
        return 1
    }
    return 0
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int v1 = 0, v2 = 1;
        for(int i=0; i<num_list.length; i++){
            v1+=num_list[i];
            v2*=num_list[i];
        }
        return (v2<Math.pow(v1,2))?1:0;
    }
}